plugins/Facebook/class.facebook.plugin.php

```diff
@@ -13,7 +13,7 @@ class FacebookPlugin extends Gdn_Plugin {
     const API_VERSION = '2.7';
 
     /** Authentication table key. */
-    const ProviderKey = 'Facebook';
+    const PROVIDER_KEY = 'Facebook';
```

Comment:
I've only found references to the `ProviderKey` class constants in older social plugins: Facebook, Google Plus and Twitter. In each case, the constant is only ever used internally within the class. That isn't to say that something, somewhere out there, isn't using these class constants for other purposes. However, it feels like a *relatively* safe change.